test/cypress/integration/05-user_v2.test.js

```diff
@@ -1,26 +1,38 @@
 describe('New users profiles', () => {
   describe('Contributions section', () => {
-    it.skip('Shows contributions with date since and amount contributed', () => {
-      // TODO
+    before(() => {
+      cy.visit('/xdamman/v2');
+      cy.get('[data-cy=Contributions]');
+    });
+    it('Shows contributions with date since and amount contributed', () => {
+      cy.get('[data-cy=collectives]').should('have.length.greaterThan', 1);
+      cy.get('[data-cy=date-since]').contains('since');
+      cy.get('[data-cy=amount-contributed]');
```

Comment:
As per https://github.com/opencollective/opencollective-frontend/pull/2668#issuecomment-536454524, we also need to check the content as the DOM can be there while the content inside of it is broken.